app/src/main/java/com/waseefakhtar/doseapp/DoseApp.kt

```diff
@@ -58,6 +65,9 @@ fun DoseApp() {
             modifier = Modifier.fillMaxSize(),
             color = MaterialTheme.colorScheme.background
         ) {
+            // For the snackbar
+            var openMySnackbar by remember { mutableStateOf(false) }
```

Comment:
Let's follow the convention here: 

```suggestion
            var snackbarVisibility by remember { mutableStateOf(false) }
```